### ISSUE TITLE
Error in installing "torchac_cuda" with undefined symbol:  __nvJitLinkComplete_12_4, version libnvJitLink.so.12

### ISSUE THREAD

**Issue by kzos** [LMCache/LMCache#230]
Issue link for torchac_cuda:-

https://github.com/LMCache/torchac_cuda/issues/1

**Comment by kzos** [comment]
Create fresh conda environment with python==3.12
```
1) conda install pytorch torchvision torchaudio pytorch-cuda=12.4 -c pytorch -c nvidia
2) https://pypi.org/project/nvidia-nvjitlink-cu12/
3) export LD_LIBRARY_PATH=/nfs/scratch/miniconda3/envs/cachepy312/lib/python3.12/site-packages/nvidia/nvjitlink/lib:$LD_LIBRARY_PATH
```

fixed the issue.

**Comment by ApostaC** [comment]
Hi @kzos , thanks for submitting the issue. We had the same issue before: #180 

Also, we do have a plan to release a binary package to solve this problem in the future.

**Comment by Huixxi** [comment]
```
from lmcache.experimental.storage_backend.storage_manager import StorageManager
(VllmWorker rank=3 pid=150413) ERROR 04-24 07:04:06 [multiproc_executor.py:435] ImportError: /home/logs/huanxi/lmcache_venv_hu/lib/python3.10/site-packages/torchac_cuda.cpython-310-x86_64-linux-gnu.so:
 undefined symbol: _ZN3c106detail23torchInternalAssertFailEPKcS2_jS2_RKNSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEEE
```
Does imcache not support py310?

**Comment by ApostaC** [comment]
@Huixxi Yeah, it should support. This could be related to the issue with the torch version. We will build and release a new pip package today, but it would be better to compile the latest code in your local environment, since we are quickly updating our code.

**Comment by Monstertail** [comment]
> ```
> from lmcache.experimental.storage_backend.storage_manager import StorageManager
> (VllmWorker rank=3 pid=150413) ERROR 04-24 07:04:06 [multiproc_executor.py:435] ImportError: /home/logs/huanxi/lmcache_venv_hu/lib/python3.10/site-packages/torchac_cuda.cpython-310-x86_64-linux-gnu.so:
>  undefined symbol: _ZN3c106detail23torchInternalAssertFailEPKcS2_jS2_RKNSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEEE
> ```
> 
> imcache不支持py310吗？

The same error. Have you tried to fix this error?
